Document/0x04i-Testing-user-interaction.md

```diff
@@ -0,0 +1,36 @@
+## Testing User Education
+A lot has happened lately in terms of responsibilities that developers have to educate users on what they need to know.
+This has shifted especially with the introduction of the [General Data Protection Regulation (GDPR)](https://gdpr-info.eu/ "GDPR") in Europe. Ever since then, it is best to educate users on what is happening with their private data and why.
+Next, it is a good practice to inform the user about how he can best use the application to ensure a secure processing of his information.
+Both items will be dealt with here. Please not that this is the MSTG project and not a legal handbook. Therefore, we will not cover the GDPR and other possibly relevant laws here.
+
+
+### Informing users on their private information
+When you need personal information from a user for your business process, then the user needs to be informed what you do with the data and why you need it. If there is a third party doing the actual processing of the data, then the user should be informed about that. Lastly, there are three processes you need to support:
+- The right to be forgotten: A user needs to know how - and be be able to request the deletion of his data.
```

Comment:
Kind of a weird way of saying it, and I can't seem to fix it :p How about just the following? Or do you want to make a distinct difference between providing functionality, and informing the user on where that functionality is?

> The right to be forgotten: A user needs to be able to request the deletion of his data.